backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CreateVolumeContainerCommand.java

```diff
@@ -85,6 +86,12 @@ private VolumeType getType() {
             return getParameters().getVolumeType();
         }
 
+        if(getParameters().getBackup() == DiskBackup.Incremental) {
+            return getParameters().getVolumeType() == VolumeType.Preallocated ?
```

Comment:
I think you can just return getParameters().getVolumeType()

and you can remove lines 85-87